src/components/MovieCard/MovieCard.tsx

```diff
@@ -1,7 +1,47 @@
 import React from 'react';
-
+import { Movie } from '../../types/Movie';
 import './MovieCard.scss';
 
-export const MovieCard: React.FC = () => (
-  <>Put the card here</>
+type MovieTypes = {
```

Comment:
Make it simple, just use `Props` name everywhere
```suggestion
type Props = {
```